certbot-apache/certbot_apache/tests/augeasnode_test.py

```diff
@@ -72,3 +74,59 @@ def test_find_comments(self):
         self.assertTrue(rootcomment[0].filepath.endswith(
             "debian_apache_2_4/multiple_vhosts/apache2/apache2.conf"
         ))
+
+    def test_set_parameters(self):
+        servernames = self.config.parser_root.find_directives("servername")
+        names = []  # type: List[str]
+        for servername in servernames:
+            names += servername.parameters
+        self.assertFalse("going_to_set_this" in names)
+        servernames[0].set_parameters(["something", "going_to_set_this"])
+        servernames = self.config.parser_root.find_directives("servername")
+        names = []
+        for servername in servernames:
+            names += servername.parameters
+        self.assertTrue("going_to_set_this" in names)
+
+    def test_set_parameters_atinit(self):
+        from certbot_apache.augeasparser import AugeasDirectiveNode
+        servernames = self.config.parser_root.find_directives("servername")
+        setparam = "certbot_apache.augeasparser.AugeasDirectiveNode.set_parameters"
+        with mock.patch(setparam) as mock_set:
+            AugeasDirectiveNode(
+                name=servernames[0].name,
+                parameters=["test", "setting", "these"],
+                ancestor=assertions.PASS,
```

Comment:
didn't we decide we're not using ancestor for augeas? what if we just... stop including it?

Reply:
bumping this because I think it wasn't in the original review so may have gotten lost

Reply:
We decided to use the dummy assertion pass string as ancestor in order to raise an exception early if we mistakenly use it in the user code. It needs to be there however, as the ancestor `None` denotes the root node. So for me this looks like it's doing exactly what was decided.